src/main/java/com/scaler/conduit/security/JWTAuthenticationFilter.java

```diff
@@ -74,12 +74,12 @@ public Authentication convert(HttpServletRequest request) {
                 return null;
             }
 
-            if (!authHeader.startsWith("Token ")) {
+            if (!authHeader.startsWith("Bearer ")) {
```

Comment:
Bearer is correct as per spec, but realworld.io API had mentioned `Token` that's why it was kept Token 

I think it is ok to update it to Bearer :)

Reply:
Yes. While testing with Postman I could only do basic auth which has bearer by default.